examples/fuse_examples/tests/test_classification_cmmd.py

```diff
@@ -17,70 +17,72 @@
-        self.CACHE_PATH = tempfile.mkdtemp(prefix="cache_data")
-        # Name of the cached data folder
-        EXPERIMENT_CACHE = 'CMMD_cache'
-
-        self.paths = {'data_dir': self.ROOT_DATA,
-                      'model_dir': os.path.join(self.ROOT, EXPERIMENT, 'model_dir'),
-                      'data_misc_dir' : os.path.join(self.ROOT, 'data_misc'),
-                      'force_reset_model_dir': True,
-                      'cache_dir': os.path.join(self.CACHE_PATH, EXPERIMENT_CACHE + '_cache_dir'),
-                      'inference_dir': os.path.join(self.ROOT, EXPERIMENT, 'infer_dir'),
-                      'eval_dir': os.path.join(self.ROOT, EXPERIMENT, 'eval_dir')}
-
-        self.infer_common_params = INFER_COMMON_PARAMS
+        self.root = tempfile.mkdtemp()
+        self.cfg = NDict({'paths': 
+            {'data_dir': os.environ["CMMD_DATA_PATH"],
+             'model_dir':  os.path.join(self.root, 'model_new/InceptionResnetV2_2017_test'),
+             'inference_dir': os.path.join(self.root,'model_new/infer_dir'),
+             'eval_dir': os.path.join(self.root,'model_new/eval_dir'),
+             'cache_dir': os.path.join(self.root,'examples/CMMD_cache_dir'),
+             'data_misc_dir': os.path.join(self.root,'data_misc'),
+             'data_split_filename': 'cmmd_split.pkl'},
+            'train': {'force_reset_model_dir': True,
+                      'target': 'classification',
+                      'reset_cache': False,
+                      'num_workers': 1, 
+                      'num_folds': 5,
+                      'train_folds': [0, 1, 2],
+                      'validation_folds': [3],
+                      'batch_size': 2,
+                      'learning_rate': 0.0001,
+                      'weight_decay': 0,
+                      'resume_checkpoint_filename': None,
+                      'manager_train_params': {'num_gpus': 1, 'device': 'cuda', 'num_epochs': 100, 'virtual_batch_size': 1, 'start_saving_epochs': 10, 'gap_between_saving_epochs': 100},
+                      'manager_best_epoch_source': {'source': 'metrics.auc.macro_avg', 'optimization': 'max', 'on_equal_values': 'better'}},
+            'infer': {'infer_filename': 'validation_set_infer.gz', 'infer_folds': [4], 'target': 'classification', 'checkpoint': 'best', 'num_workers': 1}})
+        print(self.cfg)
+        
+        # Path to the stored dataset location
+        # dataset should be download from https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=70230508
+        # download requires NBIA data retriever https://wiki.cancerimagingarchive.net/display/NBIA/Downloading+TCIA+Images
+        # put on the folliwing in the main folder  - 
+        # 1. CMMD_clinicaldata_revision.csv which is a converted version of CMMD_clinicaldata_revision.xlsx 
+        # 2. folder named CMMD which is the downloaded data folder
 
-        self.eval_common_params = EVAL_COMMON_PARAMS
 
     def test_runner(self):
```

Comment:
You should add a decorator here:
`@run_in_subprocess()`

Reply:
done